tracing/tracing/src/main/java/io/helidon/tracing/SpanLifeCycleListener.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.tracing;
+
+/**
+ * Behavior of listeners notified of span life cycle events.
```

Comment:
I personally prefer descriptions more like "An X that Ys", so: "An interface whose implementations listen for events concerning {@link Span}s" or similar

Reply:
I tend to avoid repeating the specific category (interface, class, enum) that's self-evident in the line immediately after the Javadoc comment. That also insulates the comment from the occasional change between interface and class, for example. I'll reword it a bit anyway.